velox/docs/develop/operators.rst

```diff
@@ -600,7 +600,7 @@ results available before seeing all input.
   * - partitionKeys
     - Partition by columns.
   * - rowNumberColumnName
-    - Output column name for the row numbers.
+    - Optional output column name for the row numbers. If specified, the generated row numbers are returned in an output column.
```

Comment:
It might be helpful to clarify that this column appears after all the input columns.